src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -465,8 +462,8 @@ private void removeTerritoriesThatArentWorthAttacking(final List<ProTerritory> p
     for (final Iterator<ProTerritory> it = prioritizedTerritories.iterator(); it.hasNext();) {
```

Comment:
@ron-murhammer I would appreciate it if you could replace those for loops with for-each loops...
Can't see any mayor code smells, looks good...